base/settings/tests/testthat/test.loadPath_sitePFT.R

```diff
@@ -0,0 +1,35 @@
+test_that("`loadPath.sitePFT` gives no return value for file extensions other than csv and txt", {
+  settings <- list(host = "pecan")
+  path <- "base/settings.R"
+  expect_silent(loadPath.sitePFT(settings, path))
+})
+
+test_that("`loadPath.sitePFT` gives an error for file with number of columns not equal to 2", {
+  withr::with_tempfile("tf", fileext = ".csv", {
+    settings <- list(host = "pecan")
+    df <- data.frame(
+      h1 = c("1", "2", "3"),
+      h2 = c("a", "b", "c"),
+      h3 = c("d", "e", "f")
+    )
+    write.csv(df, tf, row.names = FALSE)
+    expect_error(
+      loadPath.sitePFT(settings, tf),
+      "There is a problem with reading the file. Either row number, column number is zero or your file does not have two columns."
```

Comment:
For long error messages like this it's often easier, and a bit more forgiving if someone rewords them later without updating the test, to only test some key substring:
```suggestion
      "row number, column number is zero or your file does not have two columns."
```